mqt/bench/tests/test_all.py

```diff
@@ -290,8 +292,18 @@ def test_routing():
             0,
             6,
             None,
+            "tket",
+            True,
```

Comment:
This can't be right, can it? This should be a configuration dictionary, similar to the Qiskit case. Please also check for the other test cases.
Have you actually run these tests? (I suppose they are not executed as part of the PR since this PR does not target the `main` branch).

Reply:
Yes, I did. This was actually on purpose, since those parameters only apply to the mapped layer and not the algorithm one selected in this case. Therefore, those settings should be ignored and I wanted to test that with faulty parameter values.

Reply:
While I see what you intend with this, this assumption (that compiler settings only influence the mapped layer) might not hold in the future. 
If proper type annotations where provided or if type checking would be employed as part of the project (which it should in the future), this would even result in an error.

Reply:
Adjusted the tests and opened a new issue for the type checking.